dom.bs

```diff
@@ -3821,7 +3815,7 @@ steps for each <a>exclusive <code>Text</code> node</a> <var>node</var> <a>descen
  <a>exclusive <code>Text</code> node</a>, if any.
 
  <li>Let <var>data</var> be the concatenation of the <a>data</a> of <var>node</var>'s
- <a>contiguous exclusive <code>Text</code> nodes</a> (excluding itself), in <a>tree order</a>.
```

Comment:
But we can't really change the concept given the other callsites. Prolly best to just not change this.